cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -1145,9 +1145,12 @@ def create_xblock_info(  # lint-amnesty, pylint: disable=too-many-statements
                 rules_url = settings.PROCTORING_SETTINGS.get("LINK_URLS", {}).get(
                     "online_proctoring_rules", ""
                 )
-                supports_onboarding = does_backend_support_onboarding(
-                    course.proctoring_provider
-                )
+
+                # Only call does_backend_support_onboarding if  not using an LTI proctoring provider
+                if course.proctoring_provider != 'lti_external':
+                    supports_onboarding = does_backend_support_onboarding(
```

Comment:
Does this error if the proctoring provider is `lti_external`? In that case I would expect `supports_onboarding` to be undefined, which should raise errors elsewhere in this codepath. You might want to give `supports_onboarding` a default value.